score-lang-compiler/src/main/java/org/openscore/lang/compiler/SlangCompiler.java

```diff
@@ -8,22 +8,28 @@
  */
 package org.openscore.lang.compiler;
 
-import org.openscore.lang.compiler.model.Executable;
+import org.openscore.lang.compiler.modeller.model.Executable;
 import org.openscore.lang.entities.CompilationArtifact;
 
 import java.io.Serializable;
 import java.util.Map;
 import java.util.Set;
 
-//todo: Eliya - add JavaDoc
 public interface SlangCompiler {
 
+    /**
+     * Compile a Slang source & its path to a CompilationArtifact object
```

Comment:
I would switch the & with the word 'and'.
make classes link with the `@link` javadocs property